src/lib/storage/create_iterable_from_segment.hpp

```diff
@@ -73,6 +76,15 @@ template <typename T, bool EraseSegmentType = HYRISE_DEBUG,
                                                      : EraseReferencedSegmentType::No)>
 auto create_iterable_from_segment(const ReferenceSegment& segment);
 
+template <typename T, typename Enabled, bool EraseSegmentType = HYRISE_DEBUG> // todo erase?
```

Comment:
Don't erase for measurements. We need to get an idea of the best performance first.